static/js/components/ShowClassification.jsx

```diff
@@ -1,7 +1,16 @@
 import React from "react";
 import PropTypes from "prop-types";
-import Button from "@material-ui/core/Button";
 import Tooltip from "@material-ui/core/Tooltip";
+import Chip from "@material-ui/core/Chip";
+
+import { makeStyles } from "@material-ui/core/styles";
+
+// Export to allow Candidate.jsx to use styles
+export const useSourceStyles = makeStyles((theme) => ({
```

Comment:
```suggestion
export const useStyles = makeStyles((theme) => ({
```
This seems to be copy pasted from Source styling, but we don't actually need to export this makeStyles() instance outside of this component.